src/modules/visitor-space/const/index.tsx

```diff
@@ -53,7 +53,7 @@ export const VISITOR_SPACE_QUERY_PARAM_CONFIG = {
 	// Filters
 	format: withDefault(StringParam, VISITOR_SPACE_QUERY_PARAM_INIT.format),
 	[SEARCH_QUERY_KEY]: ArrayParam,
-	[VisitorSpaceFilterId.Maintainer]: StringParam,
+	[VisitorSpaceFilterId.Maintainer]: withDefault(StringParam, ''),
```

Comment:
why is this needed? i would think we just omit the maintainer query param if it is empty?